superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/DefaultValue.tsx

```diff
@@ -59,7 +60,7 @@ const DefaultValue: FC<DefaultValueProps> = ({
   ) : (
     <SuperChart
       height={32}
-      width={formFilter?.filterType === 'filter_time' ? 350 : 250}
+      width={formFilter?.filterType === 'filter_time' ? 350 : INPUT_WIDTH}
```

Comment:
a bit of a bycatch - it could be nice to move `350` into `constants`, too, maybe as `TIME_FILTER_INPUT_WIDTH`.